gen.py

```diff
@@ -171,14 +244,23 @@ class Config:
 Definition = Union[ForeachDo, Stage]
 
 
+VARS_DESC = """\
+Sequence of variables for parametrization.
```

Comment:
Also reiterating a little (see my comment in `VarPath()`), we use the term "values for substitution" since they're not really variables in the strict sense (more like constants).

> That said maybe we should call them variables since the section is `vars`. This was discussed when the feature was in development though.